### ISSUE TITLE
type of ECG data

### ISSUE THREAD

**Issue by Nadja1611** [oetu/MMCL-ECG-CMR#1]
Dear Özgün,

I am just trying out your code.. I cannot really figure out what type of data I need for inference.. In your paper, you mention that you use the UK Biobank ECG data... What format does this data have ? 

Thank you for your great repo

Nadja

**Comment by oetu** [comment]
The data is stored as a torch tensor ( .pt ) with three dimensions: #subjects, #variates, and #time steps. For instance, in the case of the training data, the tensor shape is (28030, 12, 5000). The corresponding labels are provided as a python list of size 28030.
